app/src/sharedTest/java/org/oppia/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -68,6 +71,13 @@ class ProfileChooserFragmentTest {
       .build()
       .inject(this)
   }
+  @Test
+  fun testProfileChooserFragment_configurationChange_checkSpanCount_spanCountVerifiedSucessfully() {
+    ActivityScenario.launch(ProfileActivity::class.java).use {
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
+      onView(withId(R.id.profile_recycler_view)).check(RecyclerViewMatcher.hasGridItemCount(4))
```

Comment:
Update this test case to make everything more readable.
```
onView(isRoot()).perform(orientationLandscape())
onView(withId(R.id.profile_recycler_view)).check(hasGridItemCount(4))
```

Reply:
Done